deployment/helm/skaha/templates/session-volumes-mounts.yaml

```diff
@@ -0,0 +1,52 @@
+{{- define "skaha.session.commonVolumeMounts" -}}
+        {{- if .Values.sessions.extraVolumes }}
+        {{- range .Values.sessions.extraVolumes }}
+        - name: {{ .name }}
+          mountPath: {{ .volumeMount.mountPath }}
+          {{- if .volumeMount.subPath }}
+          subPath: {{ .volumeMount.subPath }}
+          {{- end }}
+          {{- if .volumeMount.readOnly }}
+          readOnly: {{ .volumeMount.readOnly }}
+          {{- end }}
+          {{- if .volumeMount.mountPropagation }}
+          mountPropagation: {{ .volumeMount.mountPropagation }}
+          {{- end }}
+        {{- end }}
+        {{- end }}
+        - mountPath: "/etc/passwd"
```

Comment:
I consider the mounting of /etc/passwd and /etc/group a system requirement so should returned to the launch-*.yaml files.  The others (/cavern and /scratch) should stay here as deployers may want to change those.

Reply:
We have moved passwd and group volumes to launch files.